index.html

```diff
@@ -50,14 +51,14 @@ <h1>Lambda Times</h1>
 
   <!-- STRETCH GOAL: ONLY UNCOMMENT THIS WHEN YOU ARE FINISHED WITH THE MAIN OBJECTIVE -->
   <!-- CAROUSEL COMPONENT -->
-  <!-- <div class="carousel">
+  <div class="carousel">
     <div class="left-button"><</div>
     <img src="./assets/carousel/mountains.jpeg" />
     <img src="./assets/carousel/computer.jpeg" />
     <img src="./assets/carousel/trees.jpeg" />
     <img src="./assets/carousel/turntable.jpeg" />
     <div class="right-button">></div>
-  </div> -->
+  </div>
 
   <!-- CARDS COMPONENT -->
   <div class="cards-container">
```

Comment:
- Noble  You did amazing job on the sprint challenge. 
- You properly implemented Tabs and Cards components into the project and were able to filter out articles by selecting particular Tab. 
- Great job tackling the stretch goal and working on Carousel functionality. It was implemented properly. Not only that but You added GSAP animations and create a infinite loop carousel , nice job on that ! 🎖 
Overall:
* Tabs and Cards properly implemented (MVP)
* Good answers to questions (MVP)
* Carousel functionality implemented (Stretch)
* Animation applied (Stretch)
* Clean Code
* Fluid git workflow
Excellent job, You displayed mastery on the subject 👍